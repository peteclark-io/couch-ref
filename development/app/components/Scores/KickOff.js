'use strict'

import React from 'react';
import styles from './styles.css';

const KickOff = React.createClass({

  propTypes: {
    'kickOff': React.PropTypes.object
  },

  render: function() {
    return (
        <span className={styles['kick-off']} ref={node => (this.root = node)}>{this.props.kickOff.format("ddd Do MMM HH:mm")}</span>
    );
  }
});

export default KickOff;
