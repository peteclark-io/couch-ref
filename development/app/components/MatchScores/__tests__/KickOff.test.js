'use strict';

import React from 'react';
import KickOff from '../KickOff';
import moment from 'moment';
import renderer from 'react-test-renderer';

it('Renders ok.', () => {
   var date = moment('2011-01-04T20:00:00Z');
   const rendered = renderer.create(
      <KickOff kickOff={date} />
   );
   expect(rendered.toJSON()).toMatchSnapshot();
});
