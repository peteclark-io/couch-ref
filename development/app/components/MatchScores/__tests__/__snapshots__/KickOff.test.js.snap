exports[`test Renders ok. 1`] = `
<span
  className="kick-off">
  Tue 4th Jan 20:00
</span>
`;
